consensus/core/src/config/params.rs

```diff
@@ -309,6 +309,10 @@ pub const MAINNET_PARAMS: Params = Params {
         "kaspadns.kaspacalc.net",
         // This DNS seeder is run by supertypo
         "n-mainnet.kaspa.ws",
+        // This DNS seeder is run by -gerri-
+        "dnsseeder-kaspa-mainnet.x-con.at",
```

Comment:
I haven't verified but I presume these have been checked and that they're legit. Right?

Reply:
Gerri (x-con.at) is a contributor from Discord on `#testnet`. I don't know more. I added this from https://github.com/kaspanet/rusty-kaspa/pull/498

`kaspa-dnsseeder.net` is H@H who I can vouch for.

tn-10 became completely dysfunctional due to the lack of any DNS seeders.  Gerri brought up one on his own, I asked H@H to set up an instance.

Reply:
I am legit yes :)
Kaspa supporter since Nov 2021, Archival Node Maintainer, Community Tester
4 redundant DNS-Nodes are located in Austria, Germany, Iceland, Netherlands